### PR TITLE
ignore the ack delay in Initial and Handshake packets

### PR DESCRIPTION
**Pull request by marten-seemann**
This should have been part of #2646.

### CODE REVIEW

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -387,18 +387,21 @@ smoothed_rtt is an exponentially-weighted moving average of an endpoint's RTT
 samples, and rttvar is the endpoint's estimated variance in the RTT samples.
 
 smoothed_rtt uses path latency after adjusting RTT samples for peer-reported
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md]:
```suggestion
The calculation of smoothed_rtt uses path latency after adjusting RTT samples for```

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -387,18 +387,21 @@ smoothed_rtt is an exponentially-weighted moving average of an endpoint's RTT
...
-acknowledgement for an ack-eliciting packet to no greater than the advertised
-max_ack_delay transport parameter.  Consequently, when a peer reports an Ack
-Delay that is greater than its max_ack_delay, the delay is attributed to reasons
-out of the peer's control, such as scheduler latency at the peer or loss of
-previous ACK frames.  Any delays beyond the peer's max_ack_delay are therefore
-considered effectively part of path delay and incorporated into the smoothed_rtt
-estimate.
+host delays ({{host-delay}}).  For packets sent in the ApplicationData packet
+number space, a peer limits any delay in sending an acknowledgement for an
+ack-eliciting packet to no greater than the advertised max_ack_delay transport
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md]:
```suggestion
ack-eliciting packet to no greater than the value it advertises in the max_ack_delay transport
```

**Comment by marten-seemann** [comment]
@martinthomson I applied your suggestions. I think this PR should be marked as editorial.

**Comment by martinthomson** [comment]
The new "MUST" threw me, but it's not really new, is it?

**Comment by marten-seemann** [comment]
That one was introduced in #2646 already.

**Pull request merged**